wetterdienst/metadata/timezone.py

```diff
@@ -7,3 +7,4 @@
 class Timezone(Enum):
     GERMANY = "Europe/Berlin"
     UTC = "UTC"
+    USA = "US/Washington"
```

Comment:
USA has 3 Timezones across northern america + Alaska + Hawaii.
Why do we need a Timezone Enum?

Reply:
I <3 enums since you taught me the use ;) 

The timezone enum is used to store different used timezones neatly and make us of the enum when implementing further weather services. We need different timezones to convert data that is provided in another timezone to UTC so that ultimately all returned data has the same timezone. 

I have not yet fully taken care and got some insights on the GHCN data and thus just defined one timezone as I believe GHCN data is only provided in one timezone. It may even be UTC after all but I have to check this first.